spec/concerns/browser_compatible_spec.rb

```diff
@@ -9,6 +9,7 @@
       'Yandex Browser' => 13,
       'Opera'          => 32,
       'Firefox'        => 42,
+      'Firefox Alpha'  => 15,
```

Comment:
Layout/AlignHash: Align the elements of a hash literal if they span more than one line.